packages/calcite-components/src/components/popover/popover.scss

```diff
@@ -3,6 +3,10 @@
  *
  * These properties can be overridden using the component's tag as selector.
  *
+ * @prop --calcite-popover-background-color: Specifies the background color of the component.
+ * @prop --calcite-popover-border-color: Specifies the border color of the component.
+ * @prop --calcite-popover-corner-radius: Specifies the corner radius of the component.
+ * @prop --calcite-popover-text-color: Specifies the text color of the component.
  * @prop --calcite-popover-z-index: Sets the z-index value for the component.
```

Comment:
Sidebar: can we deprecate component-specific z-indices? We should be using [the semantic z-indices](https://developers.arcgis.com/calcite-design-system/foundations/tokens/reference/#stacking) instead to adjust different layers. cc @alisonailea @driskull @macandcheese